tests/test_pipeline_mixin.py

```diff
@@ -916,6 +916,8 @@ def parse_args_from_docstring_by_indentation(docstring):
 
 
 def compare_pipeline_args_to_hub_spec(pipeline_class, hub_spec):
+    ALLOWED_TRANSFORMERS_ONLY_ARGS = ["timeout"]
```

Comment:
Maybe over-engineering things here but I would make this list pipeline-specific

Reply:
(feel free to ignore though, we can also do it next time a "transformers"-only arg is added)

Reply:
Yeah, it shouldn't be too painful to make that change when we need to!